Formula/git-lfs.rb

```diff
@@ -27,6 +26,7 @@ def install
       ENV["GEM_HOME"] = ".gem_home"
       system "gem", "install", "ronn"
 
+      system "make", "vendor", "RM=true"
```

Comment:
In Git LFS, we remove some of the dependencies that Go takes in by default. In this context, though, they are required. To work around an error on our part, we reinstall the dependencies, and set the `rm` command to instead run `true`, making the "deletions" a no-op. We'll fix this for v2.6.1 which will obviate the need for this section of the diff.

Reply:
Could you add a short comment above this line to explain the situation and to remove this line after v2.6.1?

Reply:
@ttaylorr bump

Reply:
Sure, and sorry for the late response -- I was out of the office last week and am just catching up on older messages. I resent this as https://github.com/Homebrew/homebrew-core/pull/33619/commits/69dc151cccff16a82ee3dc8d1bed70301f730d94.